MainMenuSample/Source/Game/FreeCamera.cs

```diff
@@ -0,0 +1,105 @@
+using FlaxEngine;
+
+namespace BasicTemplate
```

Comment:
Wrong namespace, I'm guessing it's a typical "copy pasta" mistake. 
At least pasta is tasty 🍝

Reply:
I really forgot that one.